src/test/scala/edu/ie3/simona/test/common/EvTestData.scala

```diff
@@ -14,19 +14,27 @@ import java.util.UUID
 
 trait EvTestData {
   protected val evA: MockEvModel = new MockEvModel(
-    UUID.fromString("73c041c7-68e9-470e-8ca2-21fd7dbd1797"),
+    UUID.fromString("0-0-0-0-a"),
     "evA",
     Quantities.getQuantity(11d, PowerSystemUnits.KILOWATT),
     Quantities.getQuantity(11d, PowerSystemUnits.KILOWATT),
     Quantities.getQuantity(58d, PowerSystemUnits.KILOWATTHOUR),
     200,
   )
   protected val evB: MockEvModel = new MockEvModel(
-    UUID.fromString("6d7d27a1-5cbb-4b73-aecb-dfcc5a6fb22e"),
+    UUID.fromString("0-0-0-0-b"),
     "evB",
     Quantities.getQuantity(11d, PowerSystemUnits.KILOWATT),
     Quantities.getQuantity(11d, PowerSystemUnits.KILOWATT),
     Quantities.getQuantity(80d, PowerSystemUnits.KILOWATTHOUR),
     200,
   )
+  protected val evC: MockEvModel = new MockEvModel(
+    UUID.fromString("0-0-0-0-c"),
```

Comment:
I assigned some UUIDs that are recognizable here, was useful when I was debugging fyi